### PR TITLE
[Images] Pricing and Troubleshooting updates

### PR DESCRIPTION
**Pull request by dcpena**
Updated explanations on Pricing page and added error responses to Troubleshooting.

### CODE REVIEW

File: src/content/docs/images/pricing.mdx [hunk 1]
```diff
@@ -5,53 +5,79 @@ sidebar:
   order: 6
 
 ---
+import {  Plan } from "~/components"
 
-## Cloudflare Images
+<Plan type="all" />
 
-Cloudflare Images lets you store, optimize, and deliver images from one unified API. The Images product consists of three line items:
+The Free plan includes access to the transformations feature, which lets you optimize images in remote sources (like R2.
```
Comment (Oxyjun) [inline src/content/docs/images/pricing.mdx]:
```suggestion
The Free plan includes access to the transformations feature, which lets you optimize images in remote sources (like R2). 
```

File: src/content/docs/images/pricing.mdx [hunk 2]
```diff
@@ -5,53 +5,79 @@ sidebar:
...
-## Cloudflare Images
+<Plan type="all" />
 
-Cloudflare Images lets you store, optimize, and deliver images from one unified API. The Images product consists of three line items:
+The Free plan includes access to the transformations feature, which lets you optimize images in remote sources (like R2. 
 
-* Images Stored
-* Images Delivered
-* Images Transformed
+The Paid plan allows transformations, as well as includes access to storage features within Images.
```
Comment (Oxyjun) [inline src/content/docs/images/pricing.mdx]:
```suggestion
The Paid plan allows transformations, as well as access to storage features within Images.
```

File: src/content/docs/images/pricing.mdx [hunk 3]
```diff
@@ -5,53 +5,79 @@ sidebar:
...
+
+#### Example
+
+A retail website that has 1,000 original product images that get served in 5 different sizes each month. This results in 5,000 unique transformations — or a cost of $2.50 per month.
+
+### Images Stored 
+
+Storage within Images is available only with an Images Paid plan. You can purchase storage in increments of $5 for every 100,000 images stored per month.
+
+You can create predefined variants to specify how an image should be resized, such as `thumbnail` as 100x100 and hero as 1600x500.
```
Comment (Oxyjun) [inline src/content/docs/images/pricing.mdx]:
```suggestion
You can create predefined variants to specify how an image should be resized, such as `thumbnail` as 100x100 and `hero` as 1600x500.
```

Not sure if I have enough domain knowledge for this edit, feel free to ignore.

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/images/pricing.mdx (hunk 3 above)
Comment (dcpena) [reply src/content/docs/images/pricing.mdx]: You were spot on! I accidentally missed it. Thanks for the close eye!

**Comment by deannalam** [comment]
Need to edit pricing — this isn’t ready to merge yet

**Pull request merged**